src/System.Globalization/tests/CultureInfo/CultureInfoDateTimeFormat.cs

```diff
@@ -37,6 +38,44 @@ public void PosTest2()
             Assert.Equal("HH|mm", myCultureInfo.DateTimeFormat.ShortTimePattern);
         }
 
+        [Theory]
+        [InlineData("en-US", "h:mm tt")]
+        public void TestShortTimePatternLocale(string locale, string expected)
+        {
+            CultureInfo myTestCulture = new CultureInfo(locale);
+            DateTimeFormatInfo dtf = myTestCulture.DateTimeFormat;
+            Assert.Equal(expected, dtf.ShortTimePattern);
+        }
+
+        [InlineData("fi-FI", "H:mm", "H.mm")]
```

Comment:
Sorry, misunderstood the comment.  Yes, it should have a [Theory].  Thought you were saying a [Theory] with only one piece of data could be a [Fact],